windows/win_robocopy.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Corwin Brown <blakfeld@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENATION = """
+---
+module: win_robocopy
+version_added: "2.0"
+short_description: Synchronizes the contents of two directories using Robocopy.
+description:
+    - Synchronizes the contents of two directories. Under the hood this just calls out to RoboCopy, since that should be available on most modern Windows Systems.
+options:
+  src:
+    description:
+      - Source file to sync.
+    required: true
+  dest:
+    description:
+      - Destination file to sync (Will receive contents of src).
+    required: true
+  recurse:
+    description:
+      - Includes all subdirectories (Toggles the `/e` flag to RoboCopy).
+    choices:
+      - true
+      - false
+    defaults: false
+  purge:
+    description:
+      - Deletes any files/directories found in the destination that do not exist in the source (Toggles the `/purge` flag to RoboCopy).
+    choices:
+      - true
+      - false
+    defaults: false
+author: Corwin Brown
```

Comment:
please add github id as per guidelines - makes it easier for maintainers to contact you.